main.tex

```diff
@@ -0,0 +1,86 @@
+\documentclass{article}
+\usepackage[utf8]{inputenc}
+
+\title{LaTeX}
+\author{Kinga }
+\date{May 2019}
+\title{LaTeX Project}
+\author{Kinga Straszkiewicz}
+\usepackage[style=authoryear]{biblatex}
+\usepackage{ulem}
+\usepackage{tikz}
+\usepackage{tikz-qtree}
+\usepackage{tipa}
+\usepackage{phonrule}
+\addbibresource{my_biblio.bib}
+\date{April 2019}
+
+\begin{document}
+
+\maketitle\
+
+\section{Section1}
+\uwave {My interests:}
+\begin{enumerate}
+    \item the origin of language
+    \item phonology
+    \item phonetics
+    \item grammar
+    \item syntax
+    \item phonotactics
+    \item wordbuilding
+    \item programming
+    \item semantics
+\end{enumerate}
+\newline
+\section{Section2}
+\begin{IPA}
+\textipa{/"maI "neIm Iz "kIngA. "aI "aem @ "stju:d@nt @t @ jUnI"v3:s@ti @v "gdansk. "maI "Int@r@sts "Inklu:d lIn"gwIstIks, I"spES@li f@"n6l@dZi aend "graem@r, b@t "6lseU "6rIdZIn @v "laengwIdZ "aend "proUgraemIN./}
+\end{IPA}
+\vspace{3mm}
+\newline
+\begin{IPA}
+\textipa{"mam na "imjE "kjinga. "jEs\|[tEm s\|[tu"\|[dEn\|[tk\~O u\|[ni"vEr\|[s1\|[te\|[tu g\|[da\textltailn"scegO. \|[dO "mOix \|[zai\|[n\|[tErE"\|[sOva\textltailn \addspace\|[za"li\t{t\k{s}}a C\~E j\~E\|[z1kO"\|[z\|[na\|[s\|[tfO, a "\|[zvwa\k{s}\t{t\k{s}}a fO\|[nO"lOgja i grama"\|t1ka, "jak "ruv\|[niE\k{s} pOxO"\t{\|[d\|[z}E\textltailn E jE\|[n"\|[z1ka Ora\|[s prOgramO"va\textltailn E.}
```

Comment:
1. uniwersytetu - which syllable will be stressed?
2. kjinga - please make [j] a diacritic
3. również - no [ni], please transcribe the patal nasal
4. of course, eliminate periods ;)